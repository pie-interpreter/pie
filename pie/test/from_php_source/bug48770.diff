001+ This method shouldn't be called when using parent::func!
001- %unicode|string%(26) "A::func: This should work!"