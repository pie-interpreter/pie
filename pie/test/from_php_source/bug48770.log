
---- EXPECTED OUTPUT
%unicode|string%(26) "A::func: This should work!"
---- ACTUAL OUTPUT
This method shouldn't be called when using parent::func!
---- FAILED
