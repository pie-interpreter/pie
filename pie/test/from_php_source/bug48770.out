This method shouldn't be called when using parent::func!